### ISSUE TITLE
it uh isnt downgrading

### ISSUE THREAD

**Issue by taggytheargperson** [ActualMasterOogway/Roblox-Downgrader#1]
yea celery keeps saying i've been on the version even tho i have eben using this since a while ago, and i downgrsded like 3 times

**Comment by ActualMasterOogway** [comment]
Hi there,

I think Celery checks the Name of the folder where the exe file is located and that's why it's not working. Roblox Downgrader does download the older versions of the files but keep the folder Name the same (the latent version). This has to be done or else Bloxstrap starts to upgrade roblox.

TL;DR
It doesn't work with Celery

**Comment by chicken-nugget1104** [comment]
> Hi there,
> 
> I think Celery checks the Name of the folder where the exe file is located and that's why it's not working. Roblox Downgrader does download the older versions of the files but keep the folder Name the same (the latent version). This has to be done or else Bloxstrap starts to upgrade roblox.
> 
> TL;DR It doesn't work with Celery

Just seeing this, i thought this was made FOR celery?
A part from the readme: "but the executor (for example [Celery](https://celery.zip/)) you're using hasn't."
Also: 
![image](https://github.com/user-attachments/assets/e75d1cca-3712-4fce-870a-2922264ca206)

**Comment by chicken-nugget1104** [comment]
@ActualMasterOogway Just pinging you so you see all my points i just made.

**Comment by ActualMasterOogway** [comment]
> > Hi there,
> > I think Celery checks the Name of the folder where the exe file is located and that's why it's not working. Roblox Downgrader does download the older versions of the files but keep the folder Name the same (the latent version). This has to be done or else Bloxstrap starts to upgrade roblox.
> > TL;DR It doesn't work with Celery
> 
> Just seeing this, i thought this was made FOR celery? A part from the readme: "but the executor (for example [Celery](https://celery.zip/)) you're using hasn't." Also: ![image](https://private-user-images.githubusercontent.com/160080970/363516061-e75d1cca-3712-4fce-870a-2922264ca206.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUyMjEzODEsIm5iZiI6MTcyNTIyMTA4MSwicGF0aCI6Ii8xNjAwODA5NzAvMzYzNTE2MDYxLWU3NWQxY2NhLTM3MTItNGZjZS04NzBhLTI5MjIyNjRjYTIwNi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkwMVQyMDA0NDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iZmUyZDZjODM4NGMwZWE5Njc3YzAxYTY5ZGFiMmEzOTBkNzBhMDhhNGM4MDRlYzM2ZjE2MWVmZDlhOGQyNGFjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.O8Nj_G9IK0pzCwhjcbN8HZ2OQCx9LFUuUL2HCsEjAhU)

This was made for Krampus and not celery, it worked fine on krampus but when it shutdown the first executor I thought of was celery and that's why I changed it in the readme. At the time I haven't even tested it yet with celery. I'll change the README as soon as I can but for now we hope that celery fixes this "issue" (Maybe it was intended, who knows)

**Comment by chicken-nugget1104** [comment]
> > > Hi there,
> > > I think Celery checks the Name of the folder where the exe file is located and that's why it's not working. Roblox Downgrader does download the older versions of the files but keep the folder Name the same (the latent version). This has to be done or else Bloxstrap starts to upgrade roblox.
> > > TL;DR It doesn't work with Celery
> > 
> > 
> > Just seeing this, i thought this was made FOR celery? A part from the readme: "but the executor (for example [Celery](https://celery.zip/)) you're using hasn't." Also: ![image](https://private-user-images.githubusercontent.com/160080970/363516061-e75d1cca-3712-4fce-870a-2922264ca206.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUyMjEzODEsIm5iZiI6MTcyNTIyMTA4MSwicGF0aCI6Ii8xNjAwODA5NzAvMzYzNTE2MDYxLWU3NWQxY2NhLTM3MTItNGZjZS04NzBhLTI5MjIyNjRjYTIwNi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkwMVQyMDA0NDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iZmUyZDZjODM4NGMwZWE5Njc3YzAxYTY5ZGFiMmEzOTBkNzBhMDhhNGM4MDRlYzM2ZjE2MWVmZDlhOGQyNGFjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.O8Nj_G9IK0pzCwhjcbN8HZ2OQCx9LFUuUL2HCsEjAhU)
> 
> This was made for Krampus and not celery, it worked fine on krampus but when it shutdown the first executor I thought of was celery and that's why I changed it in the readme. At the time I haven't even tested it yet with celery. I'll change the README as soon as I can but for now we hope that celery fixes this "issue" (Maybe it was intended, who knows)
 
Well i made a PR removing the "for example celery" part. #2
